tests/Microsoft.Identity.Web.Test/ServiceCollectionExtensionsTests.cs

```diff
@@ -21,9 +21,10 @@ public void AddTokenAcquisition_AddsWithCorrectLifetime()
             var services = new ServiceCollection();
 
             services.AddTokenAcquisition();
+            ServiceDescriptor[] orderedServices = services.OrderBy(s => s.ServiceType.FullName).ToArray();
```

Comment:
This is to help debugging this test, when the services evolve.

Reply:
nice!